### PR TITLE
qx.ui.layout.HBox: Local variable gaps is in the wrong scope.

### PR DESCRIPTION
**Pull request by lucapivato**

### CODE REVIEW

**Review by cboulanger — APPROVED** [review]

File: framework/source/class/qx/ui/layout/HBox.js [hunk 1]
```diff
@@ -314,12 +314,12 @@ qx.Class.define("qx.ui.layout.HBox",
 
 
       // Compute gaps
-      var spacing = this.getSpacing();
+      var spacing = this.getSpacing(), gaps;
```
Comment (oetiker) [inline framework/source/class/qx/ui/layout/HBox.js:317]:
```suggestion
      var spacing = this.getSpacing();
      var gaps;
```

**Review by oetiker — APPROVED** [review]
also note as far as `var` is concerned, there is no scope there ... so this change does not change anything afaik

**Review by level420 — CHANGES_REQUESTED** [review]
Even if var has no local scope, and both versions are equivalent, this code is clearer.
But please change it to what @oetiker suggested and put `var gaps;` in a separate line. Thank you!

**Review by hkollmann — APPROVED** [review]
After changes @oetiker suggested

**Comment by level420** [comment]
@lucapivato sorry. I've highjacked your PR and changed the declaration of gaps to a separate line.

**Review by level420 — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Pull request merged**